Alarmi/Alarmi/Sources/ViewControllers/SettingCallTimeViewController/CallTimeViewController.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  CallTimeViewController.swift
+//  Alarmi
+//
+//  Created by kwon ji won on 2022/07/18.
+//  Copyright © 2022 MoTe. All rights reserved.
+//
+
+import UIKit
+
+class CallTimeViewController: UIViewController {
+
+    @IBOutlet weak var startTimePicker: UIDatePicker!
+    @IBOutlet weak var endTimePicker: UIDatePicker!
+    @IBOutlet weak var startTimeTransferredLabel: UILabel!
+    @IBOutlet weak var endTimeTransferredLabel: UILabel!
+    @IBOutlet weak var myLocationTimezoneSegmentedControl: UISegmentedControl!
+    
+    let myTimeZone: TimeZone! = TimeZone(identifier: "America/Los_Angeles")
+    let parentTimeZone: TimeZone! = TimeZone(identifier: "Asia/Seoul")
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        attribute()
+    }
+    
+    private func attribute() {
+        configureNavigationBar()
+        configureBackground()
+    }
+    
+    private func configureNavigationBar() {
+        navigationController?.navigationBar.prefersLargeTitles = true
+        navigationController?.navigationBar.topItem?.title = "전화 시간"
+    }
+    
+    private func configureBackground() {
+        view.backgroundColor = .systemGray6
+    }
+    
+    @IBAction func startTimePickerAction(_ sender: UIDatePicker) {
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "a hh:mm"
+        
+        switch myLocationTimezoneSegmentedControl.selectedSegmentIndex {
+        case 0:
+            startTimePicker.timeZone = myTimeZone
+            dateFormatter.timeZone = parentTimeZone
+            let timeString = dateFormatter.string(from: sender.date)
+            startTimeTransferredLabel.text = "한국은 \(timeString)"
+        case 1:
+            startTimePicker.timeZone = parentTimeZone
+            dateFormatter.timeZone = myTimeZone
+            let timeString = dateFormatter.string(from: sender.date)
+            startTimeTransferredLabel.text = "쿠퍼티노는 \(timeString)"
+        default : break
```

Comment:
p3;

컨벤션에 따라 콜론은 왼쪽에 붙여쓰고,
위에 작성한 case 코드와의 통일성을 위해 break은 한 줄 띄워서 작성해주세요~

```suggestion
        default:
            break
```